pkgs/build-support/fetchfossil/default.nix

```diff
@@ -0,0 +1,20 @@
+{stdenv, fossil}:
+
+{name ? null, url, rev, md5 ? null, sha256 ? null}:
```

Comment:
We shouldn't support `md5` in newer functions. We're trying to get rid of it. See https://github.com/NixOS/nixpkgs/issues/4491